README.md

```diff
@@ -72,18 +83,21 @@ Notice that this system handle maps soss messages dyrectly to a JSON compatible
 
 ### More information
 
-- For more information, you can see the [demo steps](fiware/sample/demo.md),
+- For more information, you can see the [demo steps](fiware/doc/demo.md),
 and the related [video](https://drive.google.com/open?id=1w90DAPkovjwj7673d5RfOINlAAc7kWb1)
 - Also, you can have a look to the [internal design](fiware/doc/design.md)
-- For a fast configuration, you can use the [dockerfile](Dockerfile)
+- For a fast configuration, you can use the [dockerfile](Dockerfile). 
+**NOTICE**: Both when building and running the Dockerfile, you must specify the option `--network=host`, 
+because FIWARE SH tells Orion context broker its own IP in order to make subscriptions,
+and for it to work the docker IP must be the same as the host IP.
```

Comment:
The exact problem is that the docker IP is not accessible outside your computer. You could get another workaround, as routing the ip or aliasing it or putting fiware onion server in the same computer where soss is. In fact, we continue having this problem if the onion is outside of our LAN.
So maybe, I would put something as: 

*If fiware onion context broker can not reach the docker ip, you can run (or build if necessary) the docker with `--network=host` to share the network interface and make it accessible to your LAN.*

Reply:
Changed